CancerGov/_src/Scripts/NCI/Utilities/domManipulation.js

```diff
@@ -114,3 +114,24 @@ export const getCanonicalURL = (document = window.document) => document.querySel
  * @return {string}
  */
 export const getMetaURL = document => document.querySelector("meta[property='og:url']").getAttribute('content');
+
+/**
+ * On Some pages, the Page Options block is manually moved around the DOM based on the window width.
+ * Using matchMedia keeps the JS in sync with the CSS in a way that window.width does not.
+ */
+export const pageOptionsTransporter = () => {
+	// Page Options is manually moved on resize. Ugh.
+	const mediaQueryListener = window.matchMedia('(max-width: 1024px)');
+	const mqEventHandler = e => {
+		if(e.matches){
+			$("#PageOptionsControl1").appendTo("#blogPageOptionsInnerContainer");
+		}
+		else {
+			$("#PageOptionsControl1").appendTo("#blogPageOptionsOuterContainer");
+		}
+	}
+	mediaQueryListener.addListener(mqEventHandler)
+	// Initialize page options block in correct page location
+	mqEventHandler(mediaQueryListener)
```

Comment:
This tripped me up at first, but it's manually triggering the handler just once since there's no interaction to trigger it on page load. Is that correct?

Reply:
Exactly

Reply:
It does look like an ouroborus though. I'll add the comments.